### PR TITLE
Submit answer

### PR DESCRIPTION
**Pull request by cgeglio**
## What does this PR do?
- When a player clicks submit, checks to see if the answer they submitted is correct
- Displays a message over the game board telling the player if they were correct/incorrect and how their score is affected
- Makes the response message disappear after 2 seconds
- Adjusts the players score on the dom based on if the answer was correct/incorrect
- Hides the clue from the dom after it is answered
- Adds a takeTurn() method to the player class that will toggle their turn boolean
- Adds functionality to rotate between whos turn it is for the 3 players
- Makes CSS adjustments due to above changes

## Why are we doing this? Any context or related work?
Vital part of getting the game running

## Where should a reviewer start?
Test out the changes by selecting a clue and submitting an answer

https://github.com/VeeAndrade/Game-time/projects/1

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -168,18 +172,57 @@ function showGame() {
...
   let selectedCategory = clueCategories.find(category => category.id === selectedClue.categoryId)
   $('.selected-clue-category').text(`${selectedCategory.category.split(/(?=[A-Z])/).join(" ").toUpperCase()}`);
   $('.selected-clue-points').text(`${selectedClue.pointValue}`);
   $('.question').text(`${selectedClue.question}`);
 }
+
+function evaluateGuess() {
+  let response;
+  $('.answer-response').css("display", "flex");
+  if ($(".player-guess").val().toUpperCase() === selectedClue.answer.toUpperCase()) {
```
Comment (VeeAndrade) [inline src/index.js]: I like how you called on the toUpperCase method on the part of your function, it looks like you get past any barriers that would be put up when the player places any answers that are written weirdly with casing but this just makes all responses upper cased so I think that's awesome

File: src/index.js [hunk 2]
```diff
@@ -168,18 +172,57 @@ function showGame() {
...
+  } else {
+    $(".response").text(`Incorrect! \n The answer is ${selectedClue.answer}. \n You lose ${selectedClue.pointValue} points!`)
+    response = "incorrect";
+  }
+  $(".player-guess").val('');
+  calculateScore(response);
+}
+
+function calculateScore(response) {
+  let currentPlayer = players.find(player => player.turn);
```
Comment (VeeAndrade) [inline src/index.js]: This method using .find( ) is great! I can now see why you placed that method of this.turn = !this.turn which at first did not make sense but I can see that it rotates the turns around with the array of players.

File: src/index.js [hunk 3]
```diff
@@ -168,18 +172,57 @@ function showGame() {
...
+  if (response === 'correct') {
+    currentPlayer.score += selectedClue.pointValue;
+  } else {
+    currentPlayer.score -= selectedClue.pointValue;
+  }
+  updatePlayerScore();
+  $(`#${selectedClue.id}`).css("visibility", "hidden");
+  setTimeout(function() { $('.answer-response').css("display", "none")}, 2000);
```
Comment (VeeAndrade) [inline src/index.js]: Nice adding a set timeout method to do things in good timing. I'll be excited to see that play out

File: src/index.js [hunk 4]
```diff
@@ -168,18 +172,57 @@ function showGame() {
...
+  setTimeout(function () { switchPlayer(currentPlayer); }, 2000);
+}
+
+function switchPlayer(player) {
+  player.takeTurn();
+  let i = players.indexOf(player);
+  $(`.player${i + 1}-sidebar`).css("background-color", "transparent");
+  if (players[players.indexOf(player) + 1]) {
```
Comment (VeeAndrade) [inline src/index.js]: This is a little hard to follow, I know that if you set that part inside brackets to a variable and then call the variable with-in the brackets it would make it a little easier to read but other than that, this function looks super solid.

File: src/index.js [hunk 5]
```diff
@@ -168,18 +172,57 @@ function showGame() {
...
+    players[players.indexOf(player) + 1].takeTurn();
+    $(`.player${i + 2}-sidebar`).css("background-color", "#88A5E9");
+  } else {
```
Comment (VeeAndrade) [inline src/index.js]: Actually a little confused at this else statement, could you elaborate on that....I don't know why there would be a use case for making the player in index 0 take a turn again. I might just be missing something in here

Comment (cgeglio) [reply src/index.js]: After player 3 takes their turn, you want to go back to the first player (player at index 0). This function just starts the turn rotation back at the first player. Does that make sense?
